course-mw/course-actors-common/src/main/java/org/sunbird/learner/actors/coursebatch/dao/impl/UserCoursesDaoImpl.java

```diff
@@ -183,6 +183,9 @@ public Map<String, Object> getBatchParticipantsByPage(RequestContext requestCont
     String previousPageId = null;
     int currentOffSet = 1;
     String currentPagingState = null;
+    Response countResponse = cassandraOperation.getCountOfRecordByIdentifier(requestContext, KEYSPACE_NAME,
```

Comment:
It's not a good idea to do count(*)
let's pass the userId as fields so that the query would be like -> select count(userid)...